src/TConsole.cpp

```diff
@@ -267,6 +268,14 @@ void TConsole::Command_Clear(const std::string&, const std::vector<std::string>&
     mCommandline->write("\x1b[;H\x1b[2J");
 }
 
+void TConsole::Command_Version(const std::string& cmd, const std::vector<std::string>& args) {
+    if (!EnsureArgsCount(args, 0)) {
+        return;
+    }
+
+    Application::Console().WriteRaw("Current version: " + Application::ServerVersionString());
```

Comment:
could we get a:
```suggestion
    Application::Console().WriteRaw("Current version: v" + Application::ServerVersionString());
```
that way its consistent with the other version printouts